modules/stanford_profile_styles/lib/scss/paragraph-types/spacer/_spacer.scss

```diff
@@ -0,0 +1,9 @@
+@charset 'UTF-8';
+
+.content {
+  .paragraph-item {
+    &.ptype-stanford-spacer {
+      @include margin(0);
+    }
+  }
+}
```

Comment:
Does it need to be this specific? Can we not get away with just

```
.ptype-stanford-spacer {
  @include margin(0);
}
```

Reply:
yeah its gotta be that specific. we have `.content .paragraph-item` selector adding margin already. so its gotta be one more specific

Reply:
Ill find out.

Reply:
Ok, the combination of the media query and the nested selector make this though to change. Is it the bottom margin that you are trying to remove?

Reply:
And perhaps this should be `display: none;` on screens smaller than `md`?

Reply:
yeah. i was gonna rely on future improvement to handle that correctly

Reply:
because it has no height right now, it doesn't show anything on smaller screens.

Reply:
I don't think the bottom margin hurts at the moment so how about switching it to `display: none` on smaller screens and call it a day?